packages/php-wasm/web/src/lib/php-client.ts

```diff
@@ -184,6 +184,10 @@ export class PHPClient
 	async fileExists(path: string): Promise<boolean> {
 		return _private.get(this)!.php.fileExists(path);
 	}
+
+	async cli(command: string): Promise<string> {
+		return _private.get(this)!.php.cli(['php', '-f', '/wp-cli.phar']);
+	}
```

Comment:
We don't need this anymore because we run wp-cli differently now

Reply:
Awesome. Let's remove it from here. In another PR I might move this to `php-wasm/node` .